person.rb

```diff
@@ -25,4 +27,8 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(date, book)
+    Rental.new(date, book, self)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly change the order of parameters when creating an object of the rental as you receive person before book in the `initialize` so it will be like:
```
Rental.new(date, self, book)
```